src/Components/Admin/ObjectEditor/object.js

```diff
@@ -116,6 +132,15 @@ export default function ObjectEditor({
                 } else if (obj.value === "visual-pane") {
                     animCopy.blackboardData.jsonData.position = [0, 0, 0];
                     animCopy.blackboardData.jsonData.scaleBy = 10;
+                } else if (obj.value === "unordered-puzzle") {
+                    animCopy.blackboardData.componentType = "ordered-puzzle";
```

Comment:
This seems wrong?

If condition has: `=== "unordered-puzzle"`
But you set the componentType to `ordered-puzzle`

Reply:
In the backend, there isn't actually a "unordered-puzzle" type, it's just "ordered-puzzle" with useTargets set to False